README.md

```diff
@@ -38,29 +38,23 @@ Pistache was originally created by Mathieu Stefani (`@octal`). He continues to c
 
 The [Launchpad Team](https://launchpad.net/~pistache+team) administers the daily and stable Ubuntu pre-compiled packages.
 
-### Release Versioning
+### Versioning
 
-Please update `version.txt` accordingly with each unstable or stable release.
-
-### Interface Versioning
-
-The version of the library's public interface (ABI) is not the same as the release version. The interface version is primarily associated with the _external_ interface of the library. Different platforms handle this differently, such as AIX, GNU/Linux, and Solaris.
-
-GNU Libtool abstracts each platform's idiosyncrasies away because it is more portable than using `ar(1)` or `ranlib(1)` directly. However, it is a pain to integrate with CMake so we made do without it by setting the SONAME directly.
```

Comment:
Meson unfortunately also doesn't use libtool. To get the equivalent functionality, the user has to do [more work](https://mesonbuild.com/FAQ.html#how-do-i-do-the-equivalent-of-libtools-exportsymbol-and-exportregex). Please restore this paragraph in adapted form.

Reply:
Since I don't yet know how Libtool works I don't know what I could write there. Does this make sense?

> GNU Libtool abstracts each platform's idiosyncrasies away because it is more portable than using `ar(1)` or `ranlib(1)` directly. However, it is [not supported in Meson](https://mesonbuild.com/FAQ.html#how-do-i-do-the-equivalent-of-libtools-exportsymbol-and-exportregex) so we made do without it by setting the SONAME directly.

Reply:
Yup. That's fine.